aviary/validation_cases/benchmark_tests/test_0_iters.py

```diff
@@ -50,6 +48,10 @@ class HEZeroItersTestCase(BaseProblemPhaseTestCase):
     def test_zero_iters_height_energy(self):
         local_phase_info = deepcopy(height_energy_phase_info)
         local_inputs = deepcopy(inputs)
+        local_phase_info['pre_mission']['include_takeoff'] = True
```

Comment:
should this be post_mission since takeoff is added to the post mission?

Reply:
I vote no (see note above)

Reply:
It all may be moot if we eventually move simplified takeoff/landing into analytic phases.